src/add/operator/zip-static.ts

```diff
@@ -1,5 +1,5 @@
 import {Observable} from '../../Observable';
 import {zip} from '../../operator/zip-static';
-Observable.zip = zip;
+Observable.zip = <any>zip;
```

Comment:
Seems auto generator has function `generateNewObservableFileContents` , `generateNewOperatorFileContents ` which handles and it could be updated to handle this, but also still these file need to be included PR cause our codebase generates does file and check in into codebases, isn't it? only difference would be instead of manually modifying each files, generator should be updated and file need to be regenerated via scripts.